qiskit_nature/operators/second_quantization/qubit_converter.py

```diff
@@ -40,58 +106,235 @@ def z2symmetry_reduction(self) -> Optional[List[int]]:
+        self._set_valid(did_2_q, num_particles, z2symmetries)
 
-        return qubit_ops_reduced
+        if wrapped:
+            qubit_ops_tapered = qubit_ops_tapered[0]
+
+        return qubit_ops_tapered
+
+    def convert_more(self, second_q_ops: Union[SecondQuantizedOp, List[SecondQuantizedOp]]
+                ) -> Union[PauliSumOp, List[Optional[PauliSumOp]]]:
+        """
+        Maps the given second quantized operators to qubit operators using the same mapping,
+        reductions as was done on a preceding successful call to :meth:`convert`. This allows
+        other modules, that may build operators, to conform them to the mapping and reduction
+        that would have been  done to the problem operators.
+        but will do so only after
+
+        Args:
+            second_q_ops: A second quantized operator, or list thereof to be converted.
+
+        Returns:
+            A qubit operator or list of thereof the same length as the second_q_ops list. All
+            operators in the second_q_ops list must commute with the symmetry detected when
+            :meth:`convert` was called. If it does not then the position in the output list
+            will be set to `None` to preserve the order; or None may be directly returned
+            in the case when a single operator is provided
+
+        Raises:
+            QiskitNatureError: If property is accessed before a successful conversion
+        """
+        self._check_valid(True)
+
+        # To allow a single operator to be converted, but use the same logic that does the
+        # actual conversions, we make a single entry list of it here and unwrap to return.
+        wrapped = False
+        if isinstance(second_q_ops, SecondQuantizedOp):
+            second_q_ops = [second_q_ops]
+            wrapped = True
+
+        qubit_ops = self._map_to_qubits(second_q_ops)
+        qubit_ops_reduced, _ = self._two_qubit_reduce(qubit_ops, self.num_particles)
+        qubit_ops_tapered = self._symmetry_reduce(qubit_ops_reduced)
+
+        if wrapped:
+            qubit_ops_tapered = qubit_ops_tapered[0]
+
+        return qubit_ops_tapered
 
     def _map_to_qubits(self, second_q_ops: List[SecondQuantizedOp]) -> List[PauliSumOp]:
-        if isinstance(self._mappers, tuple):
-            raise NotImplementedError
 
         qubit_ops: List[PauliSumOp] = []
         for op in second_q_ops:
-            qubit_ops.append(self._mappers.map(op))
+            qubit_ops.append(self._mapper.map(op))
 
         return qubit_ops
 
-    def _symmetry_reduce(self, qubit_ops: List[PauliSumOp]) -> List[Optional[PauliSumOp]]:
+    def _two_qubit_reduce(self, qubit_ops: List[PauliSumOp],
+                          num_particles: Optional[Tuple[int, int]])\
+            -> Tuple[List[Optional[PauliSumOp]], bool]:
+        if self._two_qubit_reduction and not self._mapper.allows_two_qubit_reduction:
+            logger.warning("Ignoring requested two qubit reduction as mapping does not support it")
```

Comment:
```suggestion
            logger.warning("Ignoring requested two qubit reduction as mapper does not support it")
```

Reply:
Since a mapper performs a mapping either seemed fine to me - it was the mapping itself that did not support it where the mapper implemented the mapping.

Reply:
Ah because the warning could eitherwise suggest that the current implementation of the `mapper` is not capable of doing so, but the problem is that it is incompatible with the `mapping` itself. I see.